app/src/controllers/submission.ts

```diff
@@ -48,8 +48,18 @@ const controller = {
             const unitTypes = [data.singleFamilyUnits, data.multiFamilyUnits, data.multiFamilyUnits1];
             const maxUnits = unitTypes.reduce(
               (ac, value) => {
-                // Get max integer from value (eg: '1-49' returns 49)
-                const upperRange: number = value ? parseInt(value.toString().replace(/(.*)-/, '').trim()) : 0;
+                // Unit types are in the form of '1-49' or '>500'
+                // .match() with regex '/(\d+)(?!.*\d)/' matches the last number in a string, puts it in array.
+                // Get max integer from last element of array.
+                let upperRange: number = 0;
+                if (value) {
+                  upperRange = parseInt(
+                    value
+                      .toString()
+                      .match(/(\d+)(?!.*\d)/)
+                      ?.pop() ?? '0'
```

Comment:
Consideration: While not major, using an array index to get the value out is slightly more performant than using the `pop()` method, especially when the output is to be a known structure/array assuming there is a match.
- `array[index]` x 33.83 ops/sec ±2.46%
- `array.pop()` x 31.80 ops/sec ±3.96%
- Ref: https://www.measurethat.net/Benchmarks/Show/23066/0/arraypop-vs-arrayindex